python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -129,6 +129,32 @@ def MmDagModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 4), tu.rand(4, 4))
 
 
+# ==============================================================================
+
+class ContainsIntList(torch.nn.Module):
+    def __init__(self):
+        self.l = [1,2,3]
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1], torch.float32, True)
+    ])
+    def forward(self,key):
+        
+        return torch.ops.aten.__contains__(self.l, key)
```

Comment:
The reason the test is failing is that the value that gets generated for `key` is not a `torch.constant.int`. Therefore, when `key` is compared to each of the values in `self.l`, because `self.l` is composed of `torch.constant.int`s, the comparison always fails, resulting in the op not getting folded.

To make sure `key` is a `torch.constant.int` you have to hardcode the value. So do something like `torch.ops.aten.__contains__([1, 2, 3], 3)`